README.md

```diff
@@ -43,23 +43,30 @@ For more information see the [Official Documentation](https://docs.lakefs.io).
 
 ## Getting Started
 
-#### Docker
+#### Docker (MacOS, Linux)
 
-1. Ensure you have Docker installed on your computer. The MacOS and Windows installations include Docker Compose by default.
+1. Ensure you have Docker & docker-compose installed on your computer
```

Comment:
you have the same line later, "Ensure you have Docker & Docker Compose installed on your computer."
need to be consistent of how to call docker compose and if we use a period in the end.

Reply:
fixed